app/models/contact_ticket.rb

```diff
@@ -68,6 +70,10 @@ def validate_mail_name_connection
     end
   end
 
+  def email_qq
+    /[0-9]+@qq.com/.match?(email)
```

Comment:
Did you intend an exact match here?

```
irb(main):004:0> /[0-9]+@qq.com/.match? 'abc1@qq.com'
=> true
irb(main):005:0> /[0-9]+@qq.com/.match? 'abc1@qq.com.foo'
=> true
```

Reply:
good point - think it's better in 37e23a7

that regex "book" you posted in slack earlier was somewhat prescient.